formatter/src/collect.rs

```diff
@@ -10,35 +10,47 @@ use crate::{ParentIndent, ViewMacro};
 struct ViewMacroVisitor<'ast> {
     macros: Vec<ViewMacro<'ast>>,
     source: Rope,
+    format_macros: Vec<String>,
 }
 
 impl<'ast> Visit<'ast> for ViewMacroVisitor<'ast> {
     fn visit_macro(&mut self, node: &'ast Macro) {
-        if node.path.is_ident("view") {
-            let span_line = node.span().start().line;
-            let line = self.source.line(span_line - 1);
-
-            let indent_chars: Vec<_> = line
-                .chars()
-                .take_while(|&c| c == ' ' || c == '\t')
-                .collect();
-
-            let tabs = indent_chars.iter().filter(|&&c| c == '\t').count();
-            let spaces = indent_chars.iter().filter(|&&c| c == ' ').count();
-
-            if let Some(view_mac) = ViewMacro::try_parse(ParentIndent { tabs, spaces }, node) {
-                self.macros.push(view_mac);
+        for format_macro in &self.format_macros {
+            if node.path.is_ident(&format_macro) {
```

Comment:
I would rather remove the for loop:
```suggestion
        val should_format = self.format_macros.iter().any(|m| node.path.is_ident(m));
        if should_format {
            ....
```

Reply:
Makes sense